### ISSUE TITLE
A11y: Global Styles - Light Mode Warning Yellow not hitting contrast ratio standard 

### ISSUE THREAD

**Issue by rmstinson** [Esri/calcite-components#3145]
Light mode warning yellow doesn't hit contrast ratio standards on a white background. Dark mode warning yellow is fine as long as it's on the prescribed background colors. 
![Screen Shot 2021-09-30 at 3 46 43 PM](https://user-images.githubusercontent.com/12650705/135541198-cced34b2-1e76-458d-9505-15b9676da7b7.png)

Recommend we use the dark yellow `#7D7227` we use for headlines and links on the homepage as part of the out-of-the-box yellow theme option. It's kind of ugly but it's as close to yellow as we can get without it being completely brown:

![Screen Shot 2021-09-30 at 3 46 53 PM](https://user-images.githubusercontent.com/12650705/135541300-6f72b5c7-ada3-46d9-8a10-685e6b7c65f7.png)

Thoughts?

**Comment by macandcheese** [comment]
This came up in a previous issue and because the icon / accent bar color is purely decorative - and the text content itself should convey severity - it was deemed non-actionable: https://github.com/Esri/calcite-components/issues/994#issuecomment-842527676

I'll leave this issue open if others want to chime in here.

**Comment by rmstinson** [comment]
@macandcheese Gotcha! I'm wondering tho, if an icon isn't unimportant enough to not be legible for some users, should we be using the icon at all? It feels like it's not just an accessibility issue, the globe icons in the chips are barely legible to me and I don't have any visual impairments. 

![Screen Shot 2021-10-01 at 3 28 16 PM](https://user-images.githubusercontent.com/12650705/135692549-f129a6fc-4a6d-4a98-923c-64ad02cc5610.png)

It seems to me like we should reassess,  since it's the only global style we have in the set.

**Comment by macandcheese** [comment]
Yeah, maybe we could look at adjusting the yellow value (although, if the result of that is a brown-yellow, folks will likely just override it). As far as using the icon - it's already opt-in / user-overridable so it is ultimately up to the implementing design team.

> since it's the only global style we have in the set.

What do you mean by this?

**Comment by rmstinson** [comment]
> since it's the only global style we have in the set.

@macandcheese Must have lost a word in editing should be "since it's the only _inaccessible_ global style."

**Comment by rmstinson** [comment]
![Warning Icons](https://user-images.githubusercontent.com/12650705/136075117-69188d9f-4d27-4fe1-8f86-0f991526d52e.png)

Discussing it with @mitc7862 and other members of the design team, we got a lot of consensus around this direction - maintaining the yellow color theme, while keeping the warning icon as the default gray to ensure legibility - icons that are emphasizing messaging need to hit at least a 3:1 contrast ratio. 

Hopefully this strategy can banish the gold/brown direction for all time

**Comment by macandcheese** [comment]
Per conversation with @rmstinson and @mitc7862 - design work / proposal to follow.

**Comment by geospatialem** [comment]
@ashetland Can you take a look at the global style to determine the feasibility prior to our 1.0.0 release? If its feasible - once design recommendations are accepted, we can relay on to developers. cc @yelenakreyndel

**Comment by geospatialem** [comment]
@ashetland Remember touching base on this one a while back, but trying to recall the next steps as we move towards 1.0. WDYT on implementing for the January release?

**Comment by ashetland** [comment]
I believe we're going to look at this post v1.0 since it will require a rethink of color application. For now though, I believe we satisfy the AA req since the use of warning yellow is on secondary items (e.g. icons) and the "warning" itself would be written in the text. Even in the Chip example above, if the yellow Chip is meant to be a warning of some kind, that would be communicated in the chip text and the yellow border would be secondary to communicating that information.

**Comment by geospatialem** [comment]
> I believe we satisfy the AA req since the use of warning yellow is on secondary items (e.g. icons) and the "warning" itself would be written in the text. Even in the Chip example above, if the yellow Chip is meant to be a warning of some kind, that would be communicated in the chip text and the yellow border would be secondary to communicating that information.

With the above, we believe to be adhering to the a11y success criterion. @SkyeSeitz and @ashetland verified Figma documentation provides guidance for designers.

Additional research will be conducted to determine if additional documentation is needed to ensure devs do not rely on color alone, and provide additional supporting text content.

**Comment by geospatialem** [comment]
Removing the design label as research has been conducted previously.

**Comment by yelenakreyndel** [comment]
@geospatialem Aaron and I were wondering if this issue is good to be closed or if something else is needed?

**Comment by geospatialem** [comment]
@yelenakreyndel Yes, didn't comment on this sooner, but I assigned myself to determine if the documentation site needs to be modified to ensure we mention color should not be the only means to convey information on our site. Reassigned to the April milestone to tackle.

**Comment by geospatialem** [comment]
Added a section to the the accessibility page on color considerations, anticipated in next Wednesday's doc release on Wed May 10.

**Comment by geospatialem** [comment]
Verified on `next.sites` on the [accessibility page](https://developers.arcgis.com/calcite-design-system/foundations/accessibility) - targeted for release on Wed May 10.

![verify-a11y](https://user-images.githubusercontent.com/5023024/236534299-1ef11ef3-d24c-4acd-ae0d-9fa8281f0a2d.gif)
